### PR TITLE
test for action.js

### PR DESCRIPTION
**Pull request by himanshu-wedensday**
### Ticket Link
---------------------------------------------------

### Related Links
---------------------------------------------------

### Description
---------------------------------------------------

### Steps to Reproduce / Test
---------------------------------------------------

### GIF's
---------------------------------------------------

## Summary by CodeRabbit

- **Tests**
	- Introduced a test suite for validating the functionality of the `changeLocale` action creator.

### CODE REVIEW

**Comment by alichherawalla** [comment]
@coderabbitai review

File: app/components/atoms/LanguageProvider/tests/action.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { changeLocale } from '../actions';
+import { CHANGE_LOCALE } from '../constants';
+
+// Test suite for the changeLocale action creator
+describe('changeLocale action creator', () => {
+  it('should create an action to change the locale', () => {
+    const languageLocale = 'en';
```
Comment (alichherawalla) [inline app/components/atoms/LanguageProvider/tests/action.test.js:7]: does this cover all the cases that it must?

Comment (himanshu-wedensday) [reply app/components/atoms/LanguageProvider/tests/action.test.js:7]: Yes

Comment (himanshu-wedensday) [reply app/components/atoms/LanguageProvider/tests/action.test.js:7]: Screenshot 2024-04-08 at 8 42 19 PM

**Comment by himanshu-wedensday** [comment]
Simillar pr raise #72

**Pull request closed without merging**